### ISSUE TITLE
Android only crash on Hot Reload

### ISSUE THREAD

**Issue by deva666** [flutter/flutter#24710]
## Steps to Reproduce

The app crashes on every hot reload. Android only.

## Logs

```
E/DartVM  (26240): ../../third_party/dart/runtime/vm/object.h: 8954: error: Handle check failed: saw Instance of 'StartupView' expected Closure
E/DartVM  (26240): Dumping native stack trace for thread 66de
E/DartVM  (26240):   [0x00000072742d8bd0] Unknown symbol
E/DartVM  (26240):   [0x00000072742d8bd0] Unknown symbol
E/DartVM  (26240):   [0x0000007274357cb4] Unknown symbol
E/DartVM  (26240): -- End of DumpStackTrace
F/libc    (26240): Fatal signal 6 (SIGABRT), code -6 in tid 26334 (Thread-2)
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Build fingerprint: 'OnePlus/OnePlus3/OnePlus3T:8.0.0/OPR1.170623.032/1809150114:user/release-keys'
Revision: '0'
ABI: 'arm64'
pid: 26240, tid: 26334, name: Thread-2  >>> com.markodevcic.foodster <<<
signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
x0   0000000000000000  x1   00000000000066de  x2   0000000000000006  x3   0000000000000008
x4   0000000080000000  x5   0000000080000000  x6   0000000080000000  x7   0000008000000000
x8   0000000000000083  x9   3061e4e95cb81fda  x10  0000000000000000  x11  0000000000000001
x12  ffffffffffffffff  x13  0000000000000018  x14  ffffffffffffffff  x15  0000b9ddfd9424cf
x16  0000007291dcf2e8  x17  0000007291d70dfc  x18  000000726ea086f0  x19  0000000000006680
x20  00000000000066de  x21  00000072746567d0  x22  0000000000000003  x23  00000072743d0150
x24  00000072746567d8  x25  0000000000000000  x26  000000000079ffff  x27  0000007274390000
x28  0000000000000004  x29  0000007274655d30  x30  0000007291d24f24
sp   0000007274655cf0  pc   0000007291d70e04  pstate 0000000060000000
backtrace:
#00 pc 0000000000069e04  /system/lib64/libc.so (tgkill+8)
#01 pc 000000000001df20  /system/lib64/libc.so (abort+88)
#02 pc 00000000007e38d0  /data/app/com.markodevcic.foodster-7gZbrMOhCqgmCMKKiwhbtQ==/lib/arm64/libflutter.so (offset 0x6b0000)
#03 pc 0000000000e57cb4  /data/app/com.markodevcic.foodster-7gZbrMOhCqgmCMKKiwhbtQ==/lib/arm64/libflutter.so (offset 0x6b0000)
#04 pc 0000000000bd5b90  /data/app/com.markodevcic.foodster-7gZbrMOhCqgmCMKKiwhbtQ==/lib/arm64/libflutter.so (offset 0x6b0000)
#05 pc 0000000000000b48  <anonymous:0000007272e00000>
Lost connection to device.
```

## Flutter doctor
```[✓] Flutter (Channel beta, v0.11.9, on Mac OS X 10.14.1 18B75, locale en-HR)
    • Flutter version 0.11.9 at /Users/deva/Documents/Repos/flutter
    • Framework revision d48e6e433c (4 days ago), 2018-11-20 22:05:23 -0500
    • Engine revision 5c8147450d
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/deva/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[!] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    ✗ Verify that all connected devices have been paired with this computer in Xcode.
      If all devices have been paired, libimobiledevice and ideviceinstaller may require updating.
      To update with Brew, run:
        brew update
        brew uninstall --ignore-dependencies libimobiledevice
        brew uninstall --ignore-dependencies usbmuxd
        brew install --HEAD usbmuxd
        brew unlink usbmuxd
        brew link usbmuxd
        brew install --HEAD libimobiledevice
        brew install ideviceinstaller
    • ios-deploy 1.9.2
    ✗ ios-deploy out of date (1.9.4 is required). To upgrade with Brew:
        brew upgrade ios-deploy
    ! CocoaPods out of date (1.5.0 is recommended).
        CocoaPods is used to retrieve the iOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without resolving iOS dependencies with CocoaPods, plugins will not work on iOS.
        For more info, see https://flutter.io/platform-plugins
      To upgrade:
        brew upgrade cocoapods
        pod setup

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 30.0.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] IntelliJ IDEA Ultimate Edition (version 2018.2.5)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 28.0.4
    • Dart plugin version 182.4323.44

[✓] IntelliJ IDEA Community Edition (version 2017.3.4)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 22.2.2
    • Dart plugin version 173.4548.30

[✓] VS Code (version 1.29.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.20.0

[✓] Connected device (2 available)
    • ONEPLUS A3003 • a7640df2                             • android-arm64 • Android 8.0.0 (API 26)
    • iPhone XR     • DA4474DD-D419-45D4-AF66-9594A8502419 • ios           • iOS 12.1 (simulator)

```

**Comment by kangwang1988** [comment]
@deva666  
Can you give us a demo project to reproduce this problem?

**Comment by deva666** [comment]
Updated to the latest beta version. Still crashes.

```E/DartVM  ( 2271): ../../third_party/dart/runtime/vm/object.h: 8954: error: Handle check failed: saw Instance of 'StartupView' expected Closure
E/DartVM  ( 2271): Dumping native stack trace for thread ae4
E/DartVM  ( 2271):   [0x000000727409dbe0] Unknown symbol
E/DartVM  ( 2271):   [0x000000727409dbe0] Unknown symbol
E/DartVM  ( 2271):   [0x000000727411ccc4] Unknown symbol
E/DartVM  ( 2271): -- End of DumpStackTrace
F/libc    ( 2271): Fatal signal 6 (SIGABRT), code -6 in tid 2788 (Thread-2)
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Build fingerprint: 'OnePlus/OnePlus3/OnePlus3T:8.0.0/OPR1.170623.032/1809150114:user/release-keys'
Revision: '0'
ABI: 'arm64'
pid: 2271, tid: 2788, name: Thread-2  >>> com.markodevcic.foodster <<<
signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
x0   0000000000000000  x1   0000000000000ae4  x2   0000000000000006  x3   0000000000000008
x4   0000000080000000  x5   0000000080000000  x6   0000000080000000  x7   0000008000000000
x8   0000000000000083  x9   3061e4e95cb81fda  x10  0000000000000000  x11  0000000000000001
x12  ffffffffffffffff  x13  0000000000000018  x14  ffffffffffffffff  x15  0000536d37ea21fb
x16  0000007291dcf2e8  x17  0000007291d70dfc  x18  0000000000000001  x19  00000000000008df
x20  0000000000000ae4  x21  00000072709fc7d0  x22  0000000000000003  x23  0000007274195150
x24  00000072709fc7d8  x25  0000000000000000  x26  000000000079ffff  x27  0000007274155000
x28  0000000000000004  x29  00000072709fbd30  x30  0000007291d24f24
sp   00000072709fbcf0  pc   0000007291d70e04  pstate 0000000060000000
backtrace:
#00 pc 0000000000069e04  /system/lib64/libc.so (tgkill+8)
#01 pc 000000000001df20  /system/lib64/libc.so (abort+88)
#02 pc 00000000007e38d0  /data/app/com.markodevcic.foodster-GKAYXhIa-k0edy6UxajTxg==/lib/arm64/libflutter.so (offset 0x6b0000)
#03 pc 0000000000e57cc4  /data/app/com.markodevcic.foodster-GKAYXhIa-k0edy6UxajTxg==/lib/arm64/libflutter.so (offset 0x6b0000)
#04 pc 0000000000bd5ba0  /data/app/com.markodevcic.foodster-GKAYXhIa-k0edy6UxajTxg==/lib/arm64/libflutter.so (offset 0x6b0000)
#05 pc 0000000000000b48  <anonymous:0000007273200000>
```

**Comment by deva666** [comment]
I'll try to create a smaller project that replicates this, if not I can invite somebody to the project, it is hosted on Bitbucket.

**Comment by deva666** [comment]
@zoechi @kangwang1988 here's a sample that crashes
https://github.com/deva666/Flutter-Hot-Reload-Crash
It loads with some view errors, but as soon as you try to hot reload it crashes.

**Comment by deva666** [comment]
No, that is it. Just do a hot reload and it crashes.
I tried it now with a Samsung Galaxy S7 and it also crashed.

```
Syncing files to device SM G930F...
E/DartVM  ( 9006): ../../third_party/dart/runtime/vm/object.h: 8954: error: Handle check failed: saw Instance of 'StartupView' expected Closure
E/DartVM  ( 9006): Dumping native stack trace for thread 236a
E/DartVM  ( 9006):   [0x0000007933a5dbc0] Unknown symbol
E/DartVM  ( 9006):   [0x0000007933a5dbc0] Unknown symbol
E/DartVM  ( 9006):   [0x0000007933adcca4] Unknown symbol
E/DartVM  ( 9006): -- End of DumpStackTrace
F/libc    ( 9006): Fatal signal 6 (SIGABRT), code -6 in tid 9066 (Thread-5)
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Build fingerprint: 'samsung/heroltexx/herolte:8.0.0/R16NW/G930FXXS3ERHD:user/release-keys'
Revision: '8'
ABI: 'arm64'
pid: 9006, tid: 9066, name: Thread-5  >>> com.markodevcic.flutterhotreloadcrash <<<
signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
    x0   0000000000000000  x1   000000000000236a  x2   0000000000000006  x3   0000000000000008
    x4   0000000080000000  x5   0000000080000000  x6   0000000080000000  x7   0000008000000000
    x8   0000000000000083  x9   f6a3c0cef7cee6dd  x10  0000000000000000  x11  0000000000000001
    x12  ffffffffffffffff  x13  0000000000000018  x14  ffffffffffffffff  x15  0000c682cf601454
    x16  00000079520252f8  x17  0000007951fc634c  x18  0000007919e01430  x19  000000000000232e
    x20  000000000000236a  x21  0000007932c817d0  x22  0000000000000003  x23  0000007933b55150
    x24  0000007932c817d8  x25  0000000000000000  x26  000000000079ffff  x27  0000007933b15000
    x28  0000000000000004  x29  0000007932c80d30  x30  0000007951f78bd4
    sp   0000007932c80cf0  pc   0000007951fc6354  pstate 0000000060000000
backtrace:
    #00 pc 000000000006b354  /system/lib64/libc.so (tgkill+8)
    #01 pc 000000000001dbd0  /system/lib64/libc.so (abort+88)
    #02 pc 00000000007e38bc  /data/app/com.markodevcic.flutterhotreloadcrash-w7LPm8PjQhEE3HyZodXVhw==/lib/arm64/libflutter.so (offset 0x6b0000)
    #03 pc 0000000000e57ca4  /data/app/com.markodevcic.flutterhotreloadcrash-w7LPm8PjQhEE3HyZodXVhw==/lib/arm64/libflutter.so (offset 0x6b0000)
    #04 pc 0000000000bd5b80  /data/app/com.markodevcic.flutterhotreloadcrash-w7LPm8PjQhEE3HyZodXVhw==/lib/arm64/libflutter.so (offset 0x6b0000)
    #05 pc 0000000000000b48  <anonymous:0000007931640000>
Lost connection to device.
```

**Comment by rasmuschristensen** [comment]
Same situation here but for iOS. I can run flutter and HotReload from a terminal and it works, but when I use VSCode debugging (F5), make a change and save, then I get this crash on HotReload (see below).

Flutter doctor dump:
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.13.6 17G65, locale en-DK)
[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
[✓] Android Studio (version 3.2)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] VS Code (version 1.29.1)
[✓] Connected device (1 available)

**Stacktrace:**
Launching lib/main.dart on iPhone XS Max in debug mode...
Xcode build done.                                           17.4s

../../third_party/dart/runtime/vm/object.h: 8954: error: Handle check failed: saw Instance of 'HomePage' expected Closure
Dumping native stack trace for thread 2003
[0x000000010f526d14] dart::Profiler::DumpStackTrace(void*)
[0x000000010f526d14] dart::Profiler::DumpStackTrace(void*)
[0x000000010f79f222] dart::Assert::Fail(char const*, ...)
[0x000000010f578980] dart::DRT_InvokeClosureNoSuchMethod(dart::NativeArguments)
[0x0000000134980902] Unknown symbol
[0x0000000134982dc4] Unknown symbol
[0x0000000134adf8fd] Unknown symbol
[0x0000000134adf54e] Unknown symbol
[0x0000000138daa923] Unknown symbol
[0x0000000138da9e33] Unknown symbol
[0x0000000138da916d] Unknown symbol
[0x0000000138d9c603] Unknown symbol
[0x0000000139e9e20f] Unknown symbol
[0x0000000139e9df11] Unknown symbol
[0x0000000139e9de3e] Unknown symbol
[0x000000013549e839] Unknown symbol
[0x0000000139e9d653] Unknown symbol
[0x0000000139e9ca54] Unknown symbol
[0x0000000139e9c658] Unknown symbol
[0x0000000139e94559] Unknown symbol
[0x000000013549e839] Unknown symbol
[0x0000000139e9379b] Unknown symbol
[0x0000000139e91c2d] Unknown symbol
[0x0000000139e9181a] Unknown symbol
[0x000000013549e839] Unknown symbol
[0x000000013549e750] Unknown symbol
[0x0000000134980d63] Unknown symbol
[0x000000010f43046f] dart::DartEntry::InvokeFunction(dart::Function const&, dart::Array const&, dart::Array const&, unsigned long)
[0x000000010f434257] dart::DartLibraryCalls::HandleMessage(dart::Object const&, dart::Instance const&)
[0x000000010f45d926] dart::IsolateMessageHandler::HandleMessage(dart::Message*)
[0x000000010f48bd2e] dart::MessageHandler::HandleMessages(dart::MonitorLocker*, bool, bool)
[0x000000010f48be30] dart::MessageHandler::HandleNextMessage()
[0x000000010f711225] Dart_HandleMessage
[0x000000010f2fad60] tonic::DartMessageHandler::OnHandleMessage(tonic::DartState*)
[0x000000010f2fb046] std::__1::__function::__func<tonic::DartMessageHandler::OnMessage(tonic::DartState*)::$_0, std::__1::allocator<tonic::DartMessageHandler::OnMessage(tonic::DartState*)::$_0>, void ()>::operator()()
[0x000000010efee4fa] fml::MessageLoopImpl::RunExpiredTasks()
[0x000000010eff1832] fml::MessageLoopDarwin::OnTimerFire(__CFRunLoopTimer*, fml::MessageLoopDarwin*)
[0x00000001125dcf34] __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__
[0x00000001125dcb32] __CFRunLoopDoTimer
[0x00000001125dc39a] __CFRunLoopDoTimers
[0x00000001125d6a1c] __CFRunLoopRun
[0x00000001125d5e11] CFRunLoopRunSpecific
[0x00000001169fe1dd] GSEventRunModal
[0x000000011b3ce81d] UIApplicationMain
[0x000000010c8cfad0] main
[0x0000000112ec7575] start
-- End of DumpStackTrace
Lost connection to device.
Exited (sigterm)

**Comment by DanTup** [comment]
@rasmuschristensen If you set `"dart.flutterTrackWidgetCreation": false` in your VS Code settings, does the problem still occur?

**Comment by rasmuschristensen** [comment]
@DanTup yes still the same issue. I tried to flip it, no change. I also disabled it in the runner from Xcode. How does this play together with the setting in vscode?

**Comment by deva666** [comment]
I am also using VS Code, tried disabling `"dart.flutterTrackWidgetCreation"`, but still crashes

**Comment by DanTup** [comment]
Thanks for trying - it was a bit of a stab int he dark - I saw another report of a similar message that only occurred when changes were tracked, so I thought it could be a way to workaround it if the same was true here.

**Comment by rasmuschristensen** [comment]
It's a bit of a pain, as it makes debugging pretty hard

**Comment by DanTup** [comment]
@cbracken is this an engine issue? Is there anything that'd help track it down (it's somewhat outside of my knowledge, but it seems severe)

**Comment by rasmuschristensen** [comment]
@DanTup I think it started after latest dart upgrade to vscode

**Comment by rasmuschristensen** [comment]
@DanTup 
Creating a new project from vscode -> F5 -> make change -> save ->HotReload -> Crash

Launching lib/main.dart on iPhone XS Max in debug mode...
Xcode build done.                                           16.6s
flutter: *********************** AppLifecycleState.inactive
flutter: *********************** default state
flutter: *********************** AppLifecycleState.paused
flutter: *********************** default state
../../third_party/dart/runtime/vm/object.h: 8954: error: Handle check failed: saw Instance of 'MyHomePage' expected Closure
Dumping native stack trace for thread bb03
[0x000000010fdd1d14] dart::Profiler::DumpStackTrace(void*)
[0x000000010fdd1d14] dart::Profiler::DumpStackTrace(void*)
[0x000000011004a222] dart::Assert::Fail(char const*, ...)
[0x000000010fe23980] dart::DRT_InvokeClosureNoSuchMethod(dart::NativeArguments)
[0x0000000127b00902] Unknown symbol
[0x0000000127b02dc4] Unknown symbol
[0x000000012808bbd1] Unknown symbol
[0x000000012b5efe94] Unknown symbol
[0x000000012b5e62e3] Unknown symbol
[0x000000012b5e57f3] Unknown symbol
[0x000000012b5e4b2d] Unknown symbol
[0x000000012b5dadc3] Unknown symbol
[0x000000012c5ecc0f] Unknown symbol
[0x000000012c5ec911] Unknown symbol
[0x000000012c5ec83e] Unknown symbol
[0x00000001285dfbf9] Unknown symbol
[0x000000012c5ec053] Unknown symbol
[0x000000012c5eb454] Unknown symbol
[0x000000012c5eb058] Unknown symbol
[0x000000012c5da699] Unknown symbol
[0x00000001285dfbf9] Unknown symbol
[0x000000012c5d98db] Unknown symbol
[0x000000012c5d7d6d] Unknown symbol
[0x000000012c5d795a] Unknown symbol
[0x00000001285dfbf9] Unknown symbol
[0x00000001285dfb10] Unknown symbol
[0x0000000127b00d63] Unknown symbol
[0x000000010fcdb46f] dart::DartEntry::InvokeFunction(dart::Function const&, dart::Array const&, dart::Array const&, unsigned long)
[0x000000010fcdf257] dart::DartLibraryCalls::HandleMessage(dart::Object const&, dart::Instance const&)
[0x000000010fd08926] dart::IsolateMessageHandler::HandleMessage(dart::Message*)
[0x000000010fd36d2e] dart::MessageHandler::HandleMessages(dart::MonitorLocker*, bool, bool)
[0x000000010fd36e30] dart::MessageHandler::HandleNextMessage()
[0x000000010ffbc225] Dart_HandleMessage
[0x000000010fba5d60] tonic::DartMessageHandler::OnHandleMessage(tonic::DartState*)
[0x000000010fba6046] std::__1::__function::__func<tonic::DartMessageHandler::OnMessage(tonic::DartState*)::$_0, std::__1::allocator<tonic::DartMessageHandler::OnMessage(tonic::DartState*)::$_0>, void ()>::operator()()
[0x000000010f8994fa] fml::MessageLoopImpl::RunExpiredTasks()
[0x000000010f89c832] fml::MessageLoopDarwin::OnTimerFire(__CFRunLoopTimer*, fml::MessageLoopDarwin*)
[0x0000000112026f34] __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__
[0x0000000112026b32] __CFRunLoopDoTimer
[0x000000011202639a] __CFRunLoopDoTimers
[0x0000000112020a1c] __CFRunLoopRun
[0x000000011201fe11] CFRunLoopRunSpecific
[0x000000010f89c983] fml::MessageLoopDarwin::Run()
[0x000000010f89969e] fml::MessageLoopImpl::DoRun()
[0x000000010f89c07f] void* std::__1::__thread_proxy<std::__1::tuple<std::__1::unique_ptr<std::__1::__thread_struct, std::__1::default_delete<std::__1::__thread_struct> >, fml::Thread::Thread(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)::$_0> >(void*)
[0x0000000115497661] _pthread_body
Lost connection to device.
[0x000000011549750d] _pthread_body
[0x0000000115496bf9] thread_start
-- End of DumpStackTrace
Exited (sigterm)

**Comment by rasmuschristensen** [comment]
I tried installing older versions of the dart extension, but still the same (at the same time I also learned how to disable auto updates :) )
Seems like this issue is kind of the same https://github.com/flutter/flutter/issues/20217

I still have the issue when I run from vscode

**Comment by DanTup** [comment]
@rasmuschristensen Don't worry about the Flutter extension, the logic is all inside the Dart one so the version of the Flutter extension doesn't matter (for now, at least - this may change in future).

My comment was about trying an older version of Flutter itself (you can download zip files from the page I linked for all of the previous versions, and swap out your Flutter folder with it, then swap back later).

**Comment by rasmuschristensen** [comment]
Good news!!
I tried replacing flutter with this one flutter_macos_v0.11.13-beta - it worked
Replaced back to the one I had - didn't work
Downloaded latest 1.0.0 stable - It worked!!

Then I switched back to the version I had that didn't work
Then a git status......due to an issue some last week I changed tracked-widget... inside xcode_backend.sh after a comment on github somewhere. Did a git checkout on this change

Problem solved!!!!

Now everything works again and I can hotreload from vscode. Than you very much @DanTup for your time to investigate.

**Comment by DanTup** [comment]
@rasmuschristensen Great, glad you got it sorted. I guess it wasn't the same as the original issue raised here by @deva666 then so we'll leave this open.

**Comment by overgaardmorten** [comment]
Almost spent a day trying to figure this out (without luck).
I think I could use some assistance by now.

I get the following whenever hot reload is running (both Vs Code and Android Studio)

Initializing hot reload...
Syncing files to device iPhone XS...
../../third_party/dart/runtime/vm/object.h: 8954: error: Handle check failed: saw Instance of 'LoginPage' expected Closure
Dumping native stack trace for thread a407
[0x000000010d084d14] dart::Profiler::DumpStackTrace(void*)
[0x000000010d084d14] dart::Profiler::DumpStackTrace(void*)
[0x000000010d2fd222] dart::Assert::Fail(char const*, ...)
[0x000000010d0d6980] dart::DRT_InvokeClosureNoSuchMethod(dart::NativeArguments)
[0x0000000131dc0902] Unknown symbol
[0x0000000131dc2dc4] Unknown symbol
[0x00000001389957dd] Unknown symbol
[0x00000001389954cd] Unknown symbol
[0x0000000136026803] Unknown symbol
[0x0000000136025d13] Unknown symbol
[0x000000013602504d] Unknown symbol
[0x000000013601b2e3] Unknown symbol
[0x0000000136c5c6ef] Unknown symbol
[0x0000000136c5c3f1] Unknown symbol
[0x0000000136c5c31e] Unknown symbol
[0x000000013289e839] Unknown symbol
[0x0000000136c5bb33] Unknown symbol
[0x0000000136c5af34] Unknown symbol
[0x0000000136c5ab38] Unknown symbol
[0x0000000136c57219] Unknown symbol
[0x000000013289e839] Unknown symbol
[0x0000000136c5645b] Unknown symbol
[0x0000000136c548ed] Unknown symbol
[0x0000000136c544da] Unknown symbol
[0x000000013289e839] Unknown symbol
[0x000000013289e750] Unknown symbol
[0x0000000131dc0d63] Unknown symbol
[0x000000010cf8e46f] dart::DartEntry::InvokeFunction(dart::Function const&, dart::Array const&, dart::Array const&, unsigned long)
[0x000000010cf92257] dart::DartLibraryCalls::HandleMessage(dart::Object const&, dart::Instance const&)
[0x000000010cfbb926] dart::IsolateMessageHandler::HandleMessage(dart::Message*)
[0x000000010cfe9d2e] dart::MessageHandler::HandleMessages(dart::MonitorLocker*, bool, bool)
[0x000000010cfe9e30] dart::MessageHandler::HandleNextMessage()
[0x000000010d26f225] Dart_HandleMessage
[0x000000010ce58d60] tonic::DartMessageHandler::OnHandleMessage(tonic::DartState*)
[0x000000010ce59046] std::__1::__function::__func<tonic::DartMessageHandler::OnMessage(tonic::DartState*)::$_0, std::__1::allocator<tonic::DartMessageHandler::OnMessage(tonic::DartState*)::$_0>, void ()>::operator()()
[0x000000010cb4c4fa] fml::MessageLoopImpl::RunExpiredTasks()
[0x000000010cb4f832] fml::MessageLoopDarwin::OnTimerFire(__CFRunLoopTimer*, fml::MessageLoopDarwin*)
[0x000000010f30ff34] __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__
[0x000000010f30fb32] __CFRunLoopDoTimer
[0x000000010f30f39a] __CFRunLoopDoTimers
[0x000000010f309a1c] __CFRunLoopRun
[0x000000010f308e11] CFRunLoopRunSpecific
[0x000000010cb4f983] fml::MessageLoopDarwin::Run()
[0x000000010cb4c69e] fml::MessageLoopImpl::DoRun()
[0x000000010cb4f07f] void* std::__1::__thread_proxy<std::__1::tuple<std::__1::unique_ptr<std::__1::__thread_struct, std::__1::default_delete<std::__1::__thread_struct> >, fml::Thread::Thread(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)::$_0> >(void*)
[0x000000011318c33d] _pthread_body
[0x000000011318f2a7] _pthread_start
Lost connection to device.
[0x000000011318b425] thread_start
-- End of DumpStackTrace

Flutter doctor:
Mortens-MacBook-Pro:~ morten$ flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.14 18A391, locale en-DK)
[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
[✓] Android Studio (version 3.2)
[✓] VS Code (version 1.30.1)
[✓] Connected device (1 available)

• No issues found!
Mortens-MacBook-Pro:~ morten$ 

I have tried to swich from beta channel to stable - but it didnt solve it for me. Any help is appreciated

Regards Morten

**Comment by DanTup** [comment]
There's a very  similar looking issue at #25656 which was closed as no longer happening on `master`. Is anyone hitting this able to confirm if the issue goes away from them on master (you can change with `flutter channel master` temporarily and then change back).

**Comment by rasmuschristensen** [comment]
Hi @overgaardmorten 

I had the same issues. I'm also somewhere on the thread @DanTup references and you can see what we went through here https://github.com/flutter/flutter/issues/24710 . The solution that worked in the end was to make a new clone of Flutter. I think I somewhere along the way from beta to stable did some changes/fixes in the local clone of flutter. 

Have you tried to clone latest flutter stable/master and the use that instead, luckily it only required to change name of the directory where you have flutter located.

Btw. nice to see another Dane using Flutter :)

**Comment by overgaardmorten** [comment]
Hi @rasmuschristensen 
Been doing some more research.  Found the culprit - someone sitting approx 40 cm from the screen :-) I have been tweaking a little too much with the xcode project my self apparently. To get back to "normal" I created a new Flutter project from scratch - copied the "ios"-folder from the newly created project and overwrote the ios folder not working. Fixed the bundle Id. Started debugging and hot reload is working again :-)

Thanks for your help.
